browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -474,6 +474,7 @@ class MarkdownNoteDetail extends React.Component {
           value={note.content}
           storageKey={note.storage}
           noteKey={note.key}
+          isStacking={isStacking}
```

Comment:
I think you forgot to extract `isStacking` from `this.state` first. Right now it's undefined:) Also can you do the same thing to `MarkdownEditor` above this code?

Reply:
done 😌